com.microsoft.mrtk.input/Interactors/InteractorVisuals/RingReticle.cs

```diff
@@ -66,6 +67,7 @@ void Awake()
             SetReticleShrink(0);
         }
 
+        [BeforeRenderOrder(XRInteractionUpdateOrder.k_BeforeRenderLineVisual)]
```

Comment:
Maybe I've misunderstood this attribute, but doesn't it only apply to methods registered on `Application.onBeforeRender`? Which I'm not sure this one is. Just making sure we don't think this is doing something it isn't!

Reply:
You're right, it's redundant to have this marker here as well.